modules/system/console/scaffold/migration/migration.create.stub

```diff
@@ -14,8 +14,19 @@ return new class extends Migration
     public function up()
     {
         Schema::create('{{ table }}', function (Blueprint $table) {
-            $table->id();
+            $table->increments({{ primaryKey }});
+{% for field,config in fields %}
+            $table->{{ config.type }}('{{ field }}'){{ config.required == false ? '->nullable()'}}{{ config.index ? '->index()' }};
+{% endfor %}
+{% for field in jsonable %}
+            $table->text('{{ field }}')->nullable();
```

Comment:
Any thoughts on text vs mediumtext for jsonable columns? I tend to lean more towards mediumText now instead of just text

Reply:
last time I checked, text is huge already...

Reply:
You'd think, but it's not impossible to run into limitations with it. It can handle roughly 5.9k to 13k words depending on whether the characters are multibyte or not (see https://stackoverflow.com/a/35785869/6652884), but if you start adding in overhead for the JSON characters and cramming a lot of fields in there you can run out of space relatively quickly.

Reply:
@bennothommo @jaxwilko any thoughts?

Reply:
I agree with `mediumText`. I know it's probably overkill for some JSON objects, but it will mean it should cover all bases. And I don't think it has a huge ramification on storage space - it's an extra byte per row for storage.

Reply:
ok, changed